actions/llm-action-error-reporter/action.yml

```diff
@@ -154,23 +173,23 @@ runs:
       env:
         GH_TOKEN: ${{ inputs.gh-token }}
         PR_NUMBER: ${{ steps.get_pr_number.outputs.pr_number }}
-        edit_comment: ${{ inputs.edit-comment }}
+        SKIP_ON_SUCCESS: ${{ inputs.skip-on-success }}
       run: |
-        workflow_name=$(gh run view ${{ github.event.workflow_run.id }} --json workflowName --jq '.workflowName')
-        head_sha=$(gh run view ${{ github.event.workflow_run.id }} --json headSha --jq '.headSha')
-        short_sha=$(echo $head_sha | cut -c1-7)
-        repo_url=$(gh repo view --json url --jq '.url')
+        export WORKFLOW_NAME=$(gh run view ${{ github.event.workflow_run.id }} --json workflowName --jq '.workflowName')
+        export HEAD_SHA=$(gh run view ${{ github.event.workflow_run.id }} --json headSha --jq '.headSha')
+        export REPO_URL=$(gh repo view --json url --jq '.url')
+        export WF_JOB_URLS=$(gh run view ${{ github.event.workflow_run.id }} --json jobs --jq '.jobs | map("[\(.name)](\(.url))") | join(" , ")')
+        export AER_JOB_URLS=$(gh run view $GITHUB_RUN_ID --json jobs --jq '.jobs | map("[aer_workflow](\(.url))") | join(" , ")')
+        export WORKFLOW_ID=$(echo "$WORKFLOW_NAME" | tr -d ' ' | tr '[:upper:]' '[:lower:]')
 
-        pr_message="## WF: $workflow_name[#$short_sha]($repo_url/commit/$head_sha)
+        export PR_MESSAGE="
+        <$WORKFLOW_ID>
 
-        No errors found in this run. 🎉"
+        ## AER Report: [$WORKFLOW_NAME]($REPO_URL/actions/runs/${{ github.event.workflow_run.id }}) ran successfully :white_check_mark:
 
-        # Fetch the comments on the pull request and filter for comments by github.actor
-        author_comments=$(gh pr view $PR_NUMBER --json comments --jq ".comments | map(select(.author.login == \"github-actions\")) | length")
+        $AER_JOB_URLS , [commit]($REPO_URL/commit/$HEAD_SHA)
 
-        # Check if edit-comment is true and there are prior comments from github.actor
-        if [ "$edit_comment" = "true" ] && [ "$author_comments" -gt 0 ]; then
-          gh pr comment $PR_NUMBER -b "$pr_message" --edit-last
-        else
-          gh pr comment $PR_NUMBER -b "$pr_message"
-        fi
+        </$WORKFLOW_ID>"
+
+        # post the error report to the PR
+        ./error_reporter_actions/actions/llm-action-error-reporter/update_pr_comment.sh
```

Comment:
Is this path correct? Here's an example of how we can refer to the action path dynamically: 

https://github.com/smartcontractkit/.github/blob/a40ee26a986e00fbef228ded7e7da9bfed64dd2c/actions/cicd-build-publish-charts/action.yml#L93